src/interfaces/store-interfaces/state-interfaces/amalgamation-state-interface.ts

```diff
@@ -0,0 +1,15 @@
+import { AmalgamationTypes, ApprovalTypes, RelationshipTypes } from '@/enums'
+import { CourtOrderIF } from '@/interfaces'
+
+export interface AmalgamationStateIF {
+  applicationDate?: string // YYYY-MM-DD
+  approvalType: ApprovalTypes
+  approvalTypeValid: boolean
+  businessNameValid: boolean
+  courtOrder?: CourtOrderIF
+  expiry?: string // YYYY-MM-DD
+  noticeDate?: string // YYYY-MM-DD
+  relationships?: RelationshipTypes[]
+  restorationTypeValid: boolean
+  type: AmalgamationTypes
+}
```

Comment:
This is for the amalgamation-only data in the store.

Common data, such as addresses, contact info, people and shares are in their own object in the store and used by multiple filing types.